docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 
 services:
-  php:
+  service_php:
```

Comment:
I'm not a big fan of the `service_` prefix , but okay.

Reply:
What do you prefer ?

Reply:
`service_` is ok